src/Workspaces/CSharp/Portable/Formatting/FormattingHelpers.cs

```diff
@@ -341,8 +341,8 @@ public static bool IsColonInSwitchExpressionArm(this SyntaxToken token)
         public static bool IsCommaInSwitchExpression(this SyntaxToken token)
             => token.Kind() == SyntaxKind.CommaToken && token.Parent is SwitchExpressionSyntax;
 
-        public static bool IsCommaInPropertySubpattern(this SyntaxToken token)
-            => token.Kind() == SyntaxKind.CommaToken && token.Parent is PropertySubpatternSyntax;
+        public static bool IsCommaInPropertyPatternClause(this SyntaxToken token)
+            => token.Kind() == SyntaxKind.CommaToken && token.Parent is PropertyPatternClauseSyntax;
```

Comment:
Don't we have IsParentKind for this check?

Reply:
There is an extension method like that on SyntaxNode, but not on SyntaxToken.

---
In reply to: [189703875](https://github.com/dotnet/roslyn/pull/26782#discussion_r189703875) [](ancestors = 189703875)